app/presenters/hyrax/admin_set_presenter.rb

```diff
@@ -30,5 +30,17 @@ def show_path
     def available_parent_collections(*)
       []
     end
+
+    def managed_access
+      return I18n.t('hyrax.dashboard.my.collection_list.managed_access.manage') if current_ability.can?(:edit, solr_document)
+      return I18n.t('hyrax.dashboard.my.collection_list.managed_access.deposit') if current_ability.can?(:deposit, solr_document)
+      return I18n.t('hyrax.dashboard.my.collection_list.managed_access.view') if current_ability.can?(:read, solr_document)
+      ''
+    end
+
+    def allow_batch?
```

Comment:
Could you please add a comment or yard doc here describing what the cases and return values are when one has edit access?  It wasn't obvious to me from the code here but made sense when I was reading the corresponding tests.